### DISCUSSION TITLE
Review of conda-forge recipe updates

### DISCUSSION THREAD

**Discussion by bryanwweber** [executablebooks/meta#724] [General] [answered]
Hi everyone! I've created a plugin for MyST NB that I'd like to [publish to conda-forge](https://github.com/conda-forge/staged-recipes/pull/18034), but it's blocked on releasing the latest [MyST-NB on conda-forge](https://github.com/conda-forge/myst-nb-feedstock/pull/14), which is in turn blocked on the [sphinx-togglebutton recipe](https://github.com/conda-forge/sphinx-togglebutton-feedstock/pull/4). I've pinged the relevant folks (@chrisjsewell and @anirrudh) on the sphinx-togglebutton PR, but no response. I thought to come here instead, in case conda-forge notifications are too noisy. Thanks 😄

**Comment by chrisjsewell** [comment, marked as answer]
Oh yeh no worries, I thought other people were handling conda now, but can release 👍

**Reply by bryanwweber** [reply]
Thanks @chrisjsewell!

**Comment by choldgraf** [comment]
@bryanwweber are you interested in being a maintainer on any of those sub-repositories? I think that we are stretched pretty thin on all of the repositories, and if you'd be willing to help bump versions/releases/etc that would be quite helpful (to me, at least!)

**Reply by bryanwweber** [reply]
Bumping versions on Conda-forge recipes is easy enough, happy to help with as many as you need 😊
